### ISSUE TITLE
Bug: cannot recompile profile even after unloading

### ISSUE THREAD

**Issue by 3noix** [3noix/UJPS#16]
Let's consider the usual case where you run a profile, you test it, and you do a modification to the profile code. You unload the profile to recompile it (the profile object is correctly deleted), but the compilation fails (write right error). When trying to delete the dll, a message tells that the dll is still used by UJPS.exe.

It does that with Qt 5.11, but works perfectly on Qt 5.7. No Qt documentation change about this, so it must be a Qt bug.

**Comment by robert914** [comment]
Yes, I see this same behavior. I always have to shutdown UJPS in order to recompile it.

**Comment by 3noix** [comment]
... it is sad, I just did the "unload" feature to avoid that.

The problem is that I see nothing else to do than go back to a previous Qt version or wait for the next one... I see no work-around with Qt 5.11

**Comment by robert914** [comment]
Ya, I haven't had time to look at this issue. I was going to poke around in the unload to see if I could figure it out. I suspect that there is some Qt object being initialized from the DLL in 5.11 now that wasn't in 5.7 before, and it's not letting go of the DLL. I was just commenting that I do see the same problem, just to confirm it's not just your setup. :)

**Comment by 3noix** [comment]
Yes, of course, the dll is not released so it is not possible to rewrite whatever the way, from UJPS or any other program.

**Comment by 3noix** [comment]
I just installed Qt 5.15.0 and it is still a problem on this version.

**Comment by 3noix** [comment]
And with 5.15.1 too.
